releasenotes/notes/lattice_model_problem-3925bfbd2267cd4c.yaml

```diff
@@ -0,0 +1,12 @@
+---
+features:
+  - |
+    Adds a new problem class,
+    :class:`qiskit_nature.problems.second_quantization.lattice.LatticeModelProblem`,
+    for lattice models. It can create second quantized operators from a lattice model.
+    We can use the Ground State Eigensolver with it to calculate the ground state energy.
+
+    .. code-block:: python
+
+        calc = GroundStateEigensolver(converter, solver)
+        res = calc.solve(LatticeModelProblem(lattice_model))
```

Comment:
I think, if anything, this should be a fully functional snippet with all imports and variable definitions properly working. You can copy the cell from the tutorial if you want.